GitUpdateScript.sh

```diff
@@ -94,26 +94,30 @@ rm Temp_dir_log #rm removes/dels the file
 }
 
 home_dir=$(pwd)  #v1 holds the home dir
-echo "Git Repositories Found:"
```

Comment:
echo about repositories found not relevant anymore as directories as no longer being forced to the screen when creating the file